cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,161 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+== Deploying the OSToy application with Kubernetes
+
+One way to deploy the application is to have the images for the front-end and back-end microservice containers created  and stored in an image repository. You can then create Kubernetes deployments to deploy the application. The following procedures describe how to deploy the application. 
+
+=== Retrieving the login command
+
+.Procedure
```

Comment:
You might not need the `.Procedure` subheading since you already have a title right above it.